### ISSUE TITLE
4.1.0 error - no exported member RootAction

### ISSUE THREAD

**Issue by SamHH** [piotrwitek/typesafe-actions#141]
The typings supplied in `dist/` appear to be broken in `4.1.0`. They are fine in `4.0.0`.

## Steps to Reproduce
Install `4.1.0`, try to run TypeScript in strict mode. You'll receive something like this:
```
ERROR in [at-loader] ./node_modules/typesafe-actions/dist/create-reducer.d.ts:1:10
    TS2305: Module '"."' has no exported member 'RootAction'.
```

## Project Dependencies
- Typesafe-Actions Version: `4.1.0`
- TypeScript Version: `3.4.4`
- tsconfig.json:
```
{
	"compilerOptions": {
		"module": "es6",
		"allowSyntheticDefaultImports": true,
		"moduleResolution": "node",
		"target": "es2015",
		"strict": true,
		"sourceMap": true,
		"lib": ["esnext", "dom"],
		"jsx": "react",
		"baseUrl": "./src/"
	}
}
```

**Comment by foray1010** [comment]
@piotrwitek I still see this error with 4.1.1
https://circleci.com/gh/foray1010/Popup-my-Bookmarks/1930

file changes:
https://github.com/foray1010/Popup-my-Bookmarks/pull/420/files

**Comment by SamHH** [comment]
I'm also still seeing an identical error with `4.1.1`.

**Comment by piotrwitek** [comment]
The fix is to add this in the index file of your application.

```ts
declare module 'typesafe-actions' {
  export type RootAction = any;
}
```

But I recommend to check out the tutorial and learn how to set up internal type injection: https://github.com/piotrwitek/typesafe-actions#extending-internal-types-to-streamline-type-usage-with-typesafe-actions

As an alternative solution, I could add an automatic script for those who don't care about internal type injection to simply import a polyfill module in your index file:
```ts
import 'typesafe-actions/disable-type-injection';
```

**Comment by piotrwitek** [comment]
Above is only a workaround but I have found a better solution for this problem

**Comment by piotrwitek** [comment]
Closed with 38871948ce830570fbc4631f9c45028c4a74a952
